### PR TITLE
create documentation

### PR DESCRIPTION
**Pull request by GildasLepennetier**
As discussed there: https://github.com/gramps-project/Gramps.js/issues/105
Not sure if this really help

### CODE REVIEW

**Review by FlorianCassayre — COMMENTED** [review]
Never heard of this `GUNICORN_NUM_WORKERS` parameter, what system consumes this variable?

**Comment by DavidMStraub** [comment]
> Never heard of this `GUNICORN_NUM_WORKERS` parameter, what system consumes this variable?

This dockerfile:
https://github.com/gramps-project/gramps-webapi/blob/261b6dedf8732d93b7b0664e2c724d623f05fc84/Dockerfile#L59

**Comment by DavidMStraub** [comment]
@GildasLepennetier thanks for this! 

Since it's a configuration detail only specific to the Docker file, it would be more appropriate to add it to "Getting Started/Advanced Topics". Also it would be good if we can avoid having the whole docker-compose file there since it might get missed in case we change anything about the recommended compose files - the other parts of the documentation now only link to the compose files in https://github.com/gramps-project/web/tree/main/examples.

**Comment by GildasLepennetier** [comment]
> Never heard of this `GUNICORN_NUM_WORKERS` parameter, what system consumes this variable?
vous ici? :D
Yes, I still have trouble setting it on my  good old' Rpi 

> @DavidMStraub 
-> it would be more appropriate to add it to "Getting Started/Advanced Topics". 
-> avoid having the whole docker-compose file there

Makes totally sense!

**Comment by GildasLepennetier** [comment]
Hi there. I pushed two commit.
I do not know how to visualize the final result from the "mkdocs.yml"

I had to `pip install mkdocs`, since  I am under windows, add the "Script" directory to the PATH; then `pip install mkdocs-material`.
After that, `mkdocs serve` works like a charm.
Talking about charm, I added the ".idea*" to the .gitignore, to avoid syn in git the pycharm project dir.

Not sure how the next step of the branch merge should go, I guess it is a click from your side, @DavidMStraub ?

File: docs/cpu-limited.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Limit CPU usage
+
+In order to avoid high CPU/RAM usage, it is possible to set the number of workers
+using the environment variable `GUNICORN_NUM_WORKERS`.
+
+Here, we will take a number of workers = 2. Adgust it according to your needs.
```
Comment (DavidMStraub) [inline docs/cpu-limited.md:6]: Typo: adgust

**Comment by DavidMStraub** [comment]
Thanks! There is just one small typo left.

**Pull request merged**